app/webpacker/src/javascripts/cookie-manager.js

```diff
@@ -0,0 +1,127 @@
+import Cookies from 'js-cookie';
+
+export default class CookieManager {
+  constructor() {
+    this.cookiesPolicyName = 'cookies_policy';
+    this.cookiesHideConfirmName = 'cookies_preferences_set';
+    this.cookiesTreeOpenClosedDefault = 'tree_open_close_default';
+    this.expiresInOneYear = 365;
+    this.expiresInTwentyEightDays = 28;
+  }
+
+  rememberSettings() {
+    if (this.getCookiesPolicy()) {
+      const rememberSettings = this.getCookiesPolicy().remember_settings;
+
+      if (rememberSettings === 'true') {
+        return true;
+      } else if (rememberSettings === 'false') {
+        return false;
+      } else {
+        return rememberSettings;
+      }
+    } else {
+      return false;
+    }
+  }
```

Comment:
QQ - I see this pattern used extensively in your code. I'm trying to understand what other values rememberSettings could take other than boolean? It's a tad bit difficult to read imo

Personally, for easy true or false logic I'm more of a fan of the `?` operator, makes it a lil less complicating